spec/controllers/dashboard/firms_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+module Dashboard
+  RSpec.describe Dashboard::FirmsController, type: :controller do
+    let(:principal) { FactoryGirl.create(:principal) }
```

Comment:
We could drop the ``FactoryGirl``, but it would be bad idea.  The ```create``` method isn't very expressive and doesn't show up distinctly on text searches. :-1: on dropping the ```FactoryGirl```.